README.md

```diff
@@ -28,6 +28,7 @@ deploy:
         GIGALIXIR_PASSWORD: ${{ secrets.GIGALIXIR_PASSWORD }}
         GIGALIXIR_APP: my-gigalixir-app # Feel free to also put this in your secrets
         SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+        RUN_MIGRATIONS: false  # defaults to "true"
```

Comment:
```suggestion
       MIGRATIONS: false  # defaults to true
```
Lets call this `MIGRATIONS` and make the default a boolean

Reply:
Sounds good. I assumed the input would come through as a string; I wasn't sure if would be converted.